### ISSUE TITLE
/docs/editor/extension-marketplace.md has outdated screenshot

### ISSUE THREAD

**Issue by hidingmyname** [microsoft/vscode-docs#6884]
In the subsection entitled **Workspace Recommended Extensions**, the current documentation includes an obsolete image, designated as `recommendations.png`. The outdated image is as follows:

![00643_recommendations](https://github.com/microsoft/vscode-docs/assets/140602238/912ecbce-296a-47c9-a943-a10e8cb9cc6f)

The updated version should be represented by the following image:

![extensions-view-filter-menu_0](https://github.com/microsoft/vscode-docs/assets/140602238/ecd05e33-ae86-4a2c-8a9b-e468fd0766cf)

Several discrepancies are evident in the outdated screenshot:
1. The command **Extensions: Show Recommended Extensions** is no longer present in the most recent version. Indeed, the commands **Show XX Extensions** have been deprecated. These commands have been consolidated into a singular command, **View**, such as **View -> Recommended**.
![extensions-view-filter-menu_1](https://github.com/microsoft/vscode-docs/assets/140602238/ecd2ccb8-8471-4569-ac8e-6e3373f13ac2)

2. The Sort by commands have been relocated to the another icon on the extension toolbar, denoted as **Sort by -> XX**.
3. The documentation should incorporate explanations for newly introduced commands, such as **Show Running Extensions** and **Start Extension Bisect**.

In summary, the image  `recommendations.png` is outdated and requires revision to reflect the current functionalities accurately.

**Comment by ntrogh** [comment]
Thanks for raising this, @hidingmyname . Updates are underway!
